xml/bp_troubleshooting_cephadm.xml

```diff
@@ -375,5 +375,17 @@ cat config-json.json
 <screen>
 &prompt.cephuser;ceph orch daemon redeploy <replaceable>daemon</replaceable> --image=myregistry.com/<replaceable>myimage</replaceable>:<replaceable>mytag</replaceable>
 </screen>
- </sect1>
+   <important>
+    <para>
+    If you plan to run the <command>ceph-salt update</command> command after applying a PTF
+    (as described in <xref linkend="rolling-updates-running"/>), it will not
+     affect the manual changes made by the temporary fix.
+    </para>
+    <para>
+     Adversely, running the <command>ceph orch upgrade</command> command (see
```

Comment:
I question the use of the word "adversely" here. It means 

> in a way that prevents success or development; harmfully or unfavourably.

Para 1 says it won't affect the PTF changes.
The 2nd sentence of para 2 says it will upgrade any daemons the PTF installs. So what is adverse about it?

I just want to check if I am misunderstanding this.

Reply:
@lproven do we have a more formal word for 'on the other hand'?

Reply:
You could just say, "Or, run the ...."

Reply:
Yes, "conversely" would work better. 

Converse: the opposite of what was stated: Adverse: against, contrary to, hostile to. "Adverse" is negative, "converse" is not.

I think I'd just drop the word completely. "Running the `ceph orch upgrade` command..."

Reply:
coversely is best